src/RFC-0201_TariScript.md

```diff
@@ -217,9 +218,57 @@ There are two changes to the protocol data structures that must be made to allow
 The first is a relatively minor adjustment to the transaction output definition.
 The second is the inclusion of script input data and an additional public key in the transaction input field.
 
+### Commitment and public key signature
+
+The Commitment and Public Key signature ([CAPK Signature]) is used in Tari protocols as part of transaction authorization. 
+It can be thought of as a combination (not addition) of a commitment signature ([Signature on Commitment values] by F. 
+Zhang et. al. and [Commitment Signature] by G. Yu.) and a Schnorr Signature. Given a commitment 
+\\( C  = a \cdot H  + x \cdot G \\) and group element \\( PK = y \cdot G \\)\, a CAPK Signature is based on a 
+representation proof of both openings \\( (a, x) \\) and \\( y \\). It additionally binds to arbitrary message data 
+\\( m \\) via the challenge to produce a signature construction. 
+
+The construction works as follows:
+- Sample scalar nonces \\( (r_a, r_x, r_y) \\) uniformly at random.
+- Compute ephemeral values \\( C_{eph} = r_a \cdot H + r_x \cdot G \\) and \\( PK_{eph} = r_y \cdot G \\) where
+  \\( C_{eph} \\) is the ephemeral commitment and \\( PK_{eph} \\) is the ephemeral public key.
+- Use strong Fiat-Shamir to produce a challenge \\( e \\). If \\( e \overset{?}{=} 0 \\) (this is unlikely), abort and 
+  start over.
+- Compute the responses \\( u_a = r_a + e \cdot a \\) and \\( u_x = r_x + e \cdot x \\) and \\( u_y = r_y + e \cdot y\\) 
+  where \\( e \\) is the challenge scalar.
+
+A commitment signature tuple has the form \\( (u_a, u_x, C_{eph}) \\) where \\( u_a \\) and \\( u_x \\) are the first  
+second commitment signature scalars respectively. A Schnorr signature tuple has the form \\( (u_y, PK_{eph}) \\) where 
+\\( u_y \\) is the public key signature scalar. The CAPK Signature therefor has the combined form 
+\\( (u_a, u_x, C_{eph}, u_y, PK_{eph}) \\). Note that in this context a signature scalar is also a private key. 
+
+To verify:
+- The verifier computes the challenge \\( e \\) and rejects the signature if \\( e \overset{?}{=} 0 \\) (this is 
+  unlikely).
+- Verification succeeds if and only if the following equations hold:
+
+$$
+\begin{aligned}
+u_a \cdot H + u_x \cdot G &\overset{?}{=} C_{eph} + e \cdot C \\\\
+u_y \cdot G &\overset{?}{=} PK_{eph} + e \cdot PK
+\end{aligned}
+\tag{1}
+$$
+
+We note that it is possible to make verification slightly more efficient by reducing calculation of the number of 
+group elements by one. To do so, the verifier selects a nonzero scalar weight \\( w \\) uniformly at random (not 
+through Fiat-Shamir!) and accepts the signature if and only if the following equation holds:
+
+$$
+\begin{aligned}
+u_a \cdot H + (u_x + w \cdot u_y) G  - C_{eph} - w \cdot PK_{eph} - e \cdot C - (w \cdot e) PK \overset{?}{=} 0
+\end{aligned}
+\tag{2}
+$$
+
+
 ### Transaction output changes
 
-The current definition of a Tari UTXO is:
+The definition of a Tari _script-less_ UTXO would be:
```

Comment:
Please update this (and next edit) to be current and in line with a post-TariScript world.

You can't not add a script anymore, so this is outdated.